projects/microphysics/scripts/prognostic_run.py

```diff
@@ -24,6 +24,8 @@ def get_env(args):
     env["SAVE_ZARR"] = args.save_zarr
     env["SAVE_TFRECORD"] = args.save_tfrecord
     env["SAVE_NC"] = args.save_nc
+
+    env = {k: str(v) for k, v in env.items()}
```

Comment:
I was getting an error about a boolean environment variable where it was expecting a string when running the prognostic evaluation.

Reply:
same. I had to fix this in my pr too. Would be nice to keep this commit separate from the other changes.